### PR TITLE
VZ-7933 test and update VMO changes

### PR DESCRIPTION
**Pull request by knelson7**
This pull request uptakes the changes to the VMI CRD for OpenSearch and OSD.  In addition, there were some cleanup.

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -55,15 +55,13 @@ spec:
...
+            - name: OPENSEARCH_IMAGE
               value: {{ .Values.monitoringOperator.esImage }}
-            - name: ELASTICSEARCH_INIT_IMAGE
+            - name: OPENSEARCH_INIT_IMAGE
               value: {{ .Values.monitoringOperator.esInitImage }}
-            - name: KIBANA_IMAGE
+            - name: OPENSEARCH_DASHBOARDS_IMAGE
               value: {{ .Values.monitoringOperator.kibanaImage }}
-            - name: ELASTICSEARCH_WAIT_TARGET_VERSION
+            - name: OPENSEARCH_WAIT_TARGET_VERSION
```
Comment (anders-swanson) [inline platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml:64]: is this still used? I don't think ES Wait is relevant anymore

File: platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -55,15 +55,13 @@ spec:
...
             - name: ALERT_MANAGER_IMAGE
               value: {{ .Values.monitoringOperator.alertManagerImage }}
-            - name: ELASTICSEARCH_WAIT_IMAGE
-              value: {{ .Values.monitoringOperator.esWaitImage }}
-            - name: ELASTICSEARCH_IMAGE
+            - name: OPENSEARCH_IMAGE
               value: {{ .Values.monitoringOperator.esImage }}
-            - name: ELASTICSEARCH_INIT_IMAGE
+            - name: OPENSEARCH_INIT_IMAGE
               value: {{ .Values.monitoringOperator.esInitImage }}
```
Comment (anders-swanson) [inline platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml:61]: do we care about the helm keys? esInitImage, etc.

File: platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml (hunk 1 above)
Comment (Abhijithang) [reply platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml:64]: I don't think we use it, no harm in keeping it though but not required.

Comment (anders-swanson) [reply platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml:64]: let's get rid of it

Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml:64]: It is used in VMO to check if nodes are at the expected version

File: platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml (hunk 2 above)
Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano-monitoring-operator/templates/deployment.yaml:61]: I can look at cleaning that up.  I would like to get this in and do a quick followup PR.

**Review by anders-swanson — APPROVED** [review]

**Pull request merged**